beacon_node/execution_layer/src/engine_api.rs

```diff
@@ -408,8 +408,6 @@ pub struct GetPayloadResponse<T: EthSpec> {
     pub block_value: Uint256,
     #[superstruct(only(Deneb))]
     pub blobs_bundle: BlobsBundle<T>,
-    #[superstruct(only(Deneb), partial_getter(copy))]
-    pub should_override_builder: bool,
```

Comment:
This isn't a flag and is from the engine api GetPayload response, so we can't delete this one.

Reply:
Sorry my earlier comment about the comparisons probably caused some confusions here, but we should also consider this field when determine which payload to use, and this suggestion from EL (to detect censoring of transactions) isn't something VC have direct access to.

The description to this field is here
https://github.com/ethereum/execution-apis/blob/cea7eeb642052f4c2e03449dc48296def4aafc24/src/engine/cancun.md#response-2

Although I'm not sure if we keep the `ignore_builder_override_suggestion_threshold` flag, I think with this change it could complicate things a bit more and makes the behaviour less predictable if we retain this flag, keen to hear thoughts from @ethDreamer and @michaelsproul.